README.md

```diff
@@ -6,6 +6,7 @@ django-auditlog
 
 **Please remember that this app is still in development.**
 **Test this app before deploying it in production environments.**
+**We forked this repo in order to run with python 3.7, as well as make changes which improve our timeline code.**
```

Comment:
nit: our env on Heroku runs on Python 3.6 so we need to make sure we don't use any 3.7 specific APIs (I know because I did that once)